apix-integrationtests/build.gradle

```diff
@@ -5,29 +5,41 @@ plugins {
     id 'eu.xenit.alfresco' version '1.0.0' apply false
 }
 
+// Give IntelliJ a default source of non-shared classes
+ext {
+    alfresco_version = alfresco_62_version
+    alfresco_repo_version = '7.199.0'
+    alfresco_dm_version = alfresco_62_dm_version
+    alfresco_min_version = '6.2'
+    subproject_alfresco_version = 62
+}
+
 subprojects {
     apply from: "${project.projectDir}/overrides.gradle"
-    apply plugin: 'eu.xenit.de'
-    apply plugin: 'eu.xenit.alfresco'
 
     sourceSets {
         main {
             java {
-                srcDir '../../apix-impl/src/integration-test'
                 // This is needed because client side, these dependencies are not really provided
                 runtimeClasspath += configurations.compileOnly
             }
-            resources {
-                srcDir '../src/integration-test/resources'
-            }
         }
     }
 
+    dependencies {
+        compile(project(":apix-integrationtests"))
+    }
+}
+
+allprojects {
```

Comment:
Cross-configuration for all projects?
Did you want to only apply this to subprojects?
Usually configuration for all projects is only done in the root build.gradle

Reply:
I'm not the author, but I think this is copied from the `apix-impl` project tree, where it exists to make IntelliJ code navigation work in both shared and version-specific folders. Whether it's needed here I did not test.

Reply:
Its required. We could shift it to subprojects only if we put the shared source in its own subproject?

Reply:
Can we then please move `allprojects` configuration to the root build.gradle?
It should not be spread around different subprojects

Reply:
No, these settings must only be apploied to this subtree.
To illustrate:
- root
  - apix-integrationtest
   - 50
   - 51
- other stuff
It must apply to apix-integrationtest, 50, 51, ... but **not** to root or other stuff.

Reply:
Oh, I have always misunderstood what `allprojects` means. I just looked it up now.

I thought it was *all* projects in the build. But it appears to be this project + subprojects.